lib/resources/qubic_cmd_utils.dart

```diff
@@ -296,7 +296,7 @@ class QubicCmdUtils {
               value.toString(),
               tick.toString(),
               inputType.toString(),
-              payload
+              payload ?? " "
```

Comment:
@ahmed-tarek-salem is the method not accepting empty value for the payload? If that is the case, can we use "" ?

Reply:
Same as above.

Reply:
@sallymoc 

Fixed with dc48c2c, needs to be tested on desktop platforms.